vcr/issuer/issuer.go

```diff
@@ -223,7 +223,7 @@ func (i issuer) buildVC(ctx context.Context, template vc.VerifiableCredential, o
 		Issuer:            template.Issuer,
 		ExpirationDate:    template.ExpirationDate,
 		IssuanceDate:      template.IssuanceDate,
-		CredentialStatus:  template.CredentialStatus, // TODO: needs api update for this to work. do we allow external issuers to revoke this credential?
+		//CredentialStatus:  template.CredentialStatus, // not allowed for now since it requires API changes to be able to determine what status to revoke.
```

Comment:
remove

Reply:
left there intentionally so we don't activate before making sure it can be used